cluster-up/cluster/kind-1.19-sriov/provider.sh

```diff
@@ -18,9 +18,9 @@ fi
 SRIOV_TESTS_NS="${SRIOV_TESTS_NS:-kubevirt-test-default1}"
 
 function set_kind_params() {
-    export KIND_NODE_IMAGE="${KIND_NODE_IMAGE:-quay.io/kubevirtci/kindest_node:v1.17.0}"
-    export KIND_VERSION="${KIND_VERSION:-0.7.0}"
-    export KUBECTL_PATH="${KUBECTL_PATH:-/kind/bin/kubectl}"
+    export KIND_VERSION="${KIND_VERSION:-0.11.1}"
+    export KIND_NODE_IMAGE="${KIND_NODE_IMAGE:-kindest/node:v1.19.11@sha256:07db187ae84b4b7de440a73886f008cf903fcf5764ba8106a9fd5243d6f32729}"
```

Comment:
Could we use the image from quay instead of dockerhub?

Reply:
We need to use the new one that correspond to kind version https://github.com/kubernetes-sigs/kind/releases/tag/v0.11.1#user-content-new-features
I think we can mirror it to kubevirt quay registry like we did before

Reply:
OH, already mirrored and proposed the change to use it.

Reply:
Thank you :slightly_smiling_face: 
I pushed new commit to use it